recipes/libev/all/patches/0001-config-for-windows.patch

```diff
@@ -0,0 +1,145 @@
+From 69f8a65f201e132f8afd99a16ed2254bc31930a6 Mon Sep 17 00:00:00 2001
+From: Jihadist <tomasiche@gmail.com>
+Date: Sat, 31 Dec 2022 15:09:20 +0300
+Subject: [PATCH] * config for windows
+
+---
+ config.h | 128 +++++++++++++++++++++++++++++++++++++++++++++++++++++++
+ 1 file changed, 128 insertions(+)
+ create mode 100644 config.h
+
+diff --git a/config.h b/config.h
+new file mode 100644
+index 0000000..0832712
+--- /dev/null
++++ b/config.h
```

Comment:
this should also be checked in and export as a sources, there's no need to patch these additions :)
This PR is great otherwise

Reply:
Are you sure I should export as sources? Because autotools generate config.h from config.h.in for all platforms, but not for windows of course, so we used predefined config only for windows. Looks like it cannot be exported as sources https://github.com/conan-io/conan/issues/12844

Reply:
You can always export them and then conditionally use them :)